plugin/evm/validators/noop_state.go

```diff
@@ -45,7 +45,7 @@ func (n *noOpState) GetStartTime(
 	return time.Time{}, nil
 }
 
-func (n *noOpState) AddValidator(vID ids.ID, nodeID ids.NodeID, startTimestamp uint64, isActive bool) error {
+func (n *noOpState) AddValidator(vID ids.ID, nodeID ids.NodeID, weight uint64, startTimestamp uint64, isActive bool, isSoV bool) error {
```

Comment:
nit: should we just use the Validator struct here?

Reply:
done